plugins/module_utils/foreman_helper.py

```diff
@@ -26,7 +26,10 @@
 from ansible.module_utils import six
 
 try:
-    import apypie
+    try:
+        from ansible_collections.theforeman.foreman.plugins.module_utils import _apypie as apypie
+    except ImportError:
```

Comment:
this is a terrible hack to make `_foreman_spec_helper` work *outside* the ansible context (as we do it in our tests). I'd much more prefer if the tests could pretend `ansible_collections` is an importable namespace, but I'm hittin' my Python hackery limits here.

Reply:
would it work to import relative?
`like import ._apypie as apypie`

Reply:
maybe it's breaking ansible machinery...

Reply:
Ansible (pre 2.10) can't do relative imports :(